sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### 4.64.0-beta.1 (Unreleased)
 
 #### Features Added
+* Added an API to retrieve diagnostics from the change feed processor context for the all versions and deletes change feed mode. - See [PR 41738](https://github.com/Azure/azure-sdk-for-java/pull/41738)
```

Comment:
Why would this be restricted to FFCF?

Reply:
I think the plan is to first expose in FFCF, as ChangeFeedProcessorContext is not exposed for non FFCF yet. Eventually it will be added in both places

Reply:
I thought this was supposed to be the plan too. But after some discussion with Kushagra/Abhijeet I've added incremental tests as well, so since we're testing both I updated the CHANGELOG

Reply:
But you have not exposed the changeFeedProcessorContext in any of the incremental mode though? for example following is how it has exposed in FullFidelity mode:

https://github.com/Azure/azure-sdk-for-java/blob/932389b3fdaacbdd15b7d275e0592db4d5b3002d/sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ChangeFeedProcessorBuilder.java#L289

Reply:
Discussed offline, plan is Annie's first comment - WMT specifically asked for FFCF, so FFCF first and eventually incremental